libnopegl/src/pipeline_compat.h

```diff
@@ -27,7 +27,10 @@
 #include "pipeline.h"
 
 struct pipeline_compat_params {
-    const struct pipeline_params *params;
+    int type;
```

Comment:
can you add `NGLI_PIPELINE_TYPE_*` in comment?

Reply:
Done.